config/dynamic-scaling-configuration.yaml

```diff
@@ -1,12 +1,59 @@
 #This configuration file contains information about each Kafka instance types, dynamic scaling configuration
-#
-#The following properties must be defined for each instance types compute node configuration:
-#   - max_compute_nodes: the maximum number of nodes needed to create an appropriate machine pool with the configuration needed by the autoscaler
-
 ---
-developer:
-  compute_nodes_config:
-    max_compute_nodes: 3
-standard:
-  compute_nodes_config:
-    max_compute_nodes: 9
+# The version of OpenShift used during installation of newly auto created data plane clusters.
+# If the value is empty (default), then OpenShift Cluster Manager (OCM) will provision the data plane cluster using the latest version. 
+new_data_plane_openshift_version: ""
```

Comment:
Is there some reason we accept the empty value? Why not making it optional and unset meaning the default is used? and document it.

Reply:
The config is already option. If one does not set it'll default to the primitive default which is an empty string.

Reply:
:+1: . Then in this case we should change the documentation to say:
`If the value is empty or not defined, ...`

Reply:
added